### PR TITLE
Feature: add a method to manually trigger a rescan of the track

### LINKED ISSUE #97: Add a method to manually trigger a rescan of the track

### PR DESCRIPTION
**Pull request by m-braunschweig**
This PR adds buttons in the StaffUI that trigger a rescan of the track and publishes the newly scanned track to every part of the program that needs to know it.
Closes #97

### CODE REVIEW

File: src/UserInterface/StaffUI.py [hunk 1]
```diff
@@ -422,6 +424,39 @@ async def restart_system() -> Any:
...
                 return message, 200
 
+        @self.staffUI_blueprint.route('/rescan_track', methods=['POST'])
+        async def trigger_track_rescan() -> Tuple[str, int]:
+            data = await request.form
+            car = data.get('car')
+            if car is None:
+                self.logger.warning("A client attempted to start a track rescan but ")
+                return "Request didn't include a car", 400
+            vehicle = environment_mng.get_vehicle_by_vehicle_id(car)
```
Comment (iav-DaBi) [inline src/UserInterface/StaffUI.py:434]: this until line 446 should be done in the environment manager. the environment manager needs the vehicle_id

File: test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py [hunk 2]
```diff
@@ -394,3 +398,30 @@ def test_get_track_returns_from_config(initialise_dependencies):
...
     }
     assert env.get_track() is not None
+
+
+def test_track_notify():
+    """
+    Tests that the location services get notified when a new track is there due to e.g. scanning
+    """
+    config_mock = MagicMock()
+    fleet_ctrl = FleetController()
```
Comment (iav-DaBi) [inline test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py:408]: change to mock-object with spec= FleetController (please check)

File: test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py [hunk 3]
```diff
@@ -394,3 +398,30 @@ def test_get_track_returns_from_config(initialise_dependencies):
...
+    env_manager = EnvironmentManager(fleet_ctrl, configuration_handler=config_mock)
```
Comment (iav-DaBi) [inline test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py:409]: use fleet_ctrl_mock object

File: test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py (hunk 2 above)
Comment (iav-DaBi) [inline test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py:408]: change to mock-object with spec= FleetController (please check) and rename to fleet_ctrl_mock

**Review by iav-DaBi — CHANGES_REQUESTED** [review]
plz do your shit

File: src/UserInterface/StaffUI.py (hunk 1 above)
Comment (m-braunschweig) [reply src/UserInterface/StaffUI.py]: Adressed in 87b062ed334dbd25a4f98e470e0428bdfecd5773

File: test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py (hunk 2 above)
Comment (m-braunschweig) [reply test/UnitTest/EnvironmentManagement/EnvironmentManager_Test.py]: Adressed in 083c8a8c1ae29b0f5793236f27500be06d385cec

**Review by iav-DaBi — APPROVED** [review]
all clear

**Pull request merged**